src/reducers/books.js

```diff
@@ -0,0 +1,35 @@
+const newID = () => parseInt(Math.random() * 100, 10);
+const initialState = {
+  books: [
```

Comment:
requirements prescribes this code instead to be in the file "src/index.js"
![image](https://user-images.githubusercontent.com/31897434/76151912-1de5bb80-60ba-11ea-87fe-8d65a9f8b094.png)

Reply:
> requirements prescribes this code instead to be in the file "src/index.js"
> ![image](https://user-images.githubusercontent.com/31897434/76151912-1de5bb80-60ba-11ea-87fe-8d65a9f8b094.png)

You missed this. 😄